statsmodels/tsa/statespace/tests/test_representation.py

```diff
@@ -116,23 +118,25 @@ class TestClark1987Single(Clark1987):
     """
     Basic single precision test for the loglikelihood and filtered states.
     """
-    def __init__(self):
+    @classmethod
+    def setup_class(cls):
         raise SkipTest('Not implemented')
-        super(TestClark1987Single, self).__init__(
+        super(TestClark1987Single, cls).setup_class(
             dtype=np.float32, conserve_memory=0
         )
-        self.run_filter()
+        cls.run_filter()
```

Comment:
general comment:
In cases like this, I prefer the assignment to attribute to be in the basic method, i.e. `__init__` or `setup_class`
e.g. `run_filter` could return the results and setup_class could assign it to self.results.
(We still do too much attribute assignments in different methods across models, which makes it difficult to keep track of where an attribute is set.)

I haven't checked what the specific case here is and whether run_filter is also used from other places.